corehq/apps/app_manager/static/app_manager/json/commcare-profile-settings.yaml

```diff
@@ -75,14 +75,12 @@
   description: "Upload a file to serve as a CommCare home screen logo on Android phones"
```

Comment:
In the tickets it said Java and Android, here it specifically just says Android. Is this one accurate?